### PR TITLE
[SHOT-3958] Publish Plugin - Add ability to generate a .vpb file after Alias wire publish finishes

### PR DESCRIPTION
**Pull request by barbara-darkshot**
Create a new publish plugin to generate a VRED scene directly within the Alias session

### CODE REVIEW

**Review by rob-aitchison — APPROVED** [review]
This looks great to me, thanks Barbara! I pushed a commit to update the wording in the plugin description to add the VRED Pro installation requirement. Cheers!

File: hooks/tk-multi-publish2/basic/publish_vred.py [hunk 1]
```diff
@@ -0,0 +1,763 @@
...
+        # restore previous saved UI settings
+        ui_settings = settings[0]["UI Settings"]
+
+        # if no Publish template has been defined in the plugin setting, hide the publish option from the UI
+        publish_template = self.get_publish_template(settings[0], items[0])
+        if not publish_template:
+            widget.publish_to_shotgrid.hide()
+            widget.publish_to_shotgrid.setChecked(False)
+        else:
+            if "publish_to_sg" in ui_settings.keys():
```
Comment (staceyoue) [inline hooks/tk-multi-publish2/basic/publish_vred.py:241]: Suggestion: when you want to check that key exists in a dictionary or iterate over the keys in a dictionary, you can omit the call to `keys()`:
```suggestion
            if "publish_to_sg" in ui_settings:
```

File: hooks/tk-multi-publish2/basic/publish_vred.py [hunk 2]
```diff
@@ -0,0 +1,763 @@
...
+
+import alias_api
+
+HookBaseClass = sgtk.get_hook_baseclass()
+
+
+class AliasCreateVREDFilePlugin(HookBaseClass):
+    """
+    Plugin for creating a VRED scene from the current Alias session.
+    """
```
Comment (staceyoue) [inline hooks/tk-multi-publish2/basic/publish_vred.py:26]: Very minor style comment (that Ariel has suggested to me), for single line doc comments, keep the quotations in the same line:
```suggestion
class AliasCreateVREDFilePlugin(HookBaseClass):
    """Plugin for creating a VRED scene from the current Alias session."""
```

**Review by staceyoue — APPROVED** [review]
Looks good to me! Just a couple style suggestions.

**Pull request merged**